emails/2015/feb-meetup-call-for-speakers.md

```diff
@@ -0,0 +1,92 @@
+John Baldwin and I (Brian Dant) are stepping in as co-organizers for Max Nova and John Roth. Thanks John and Max for holding it together over the last 18 months or so! (If you have benefited from the contributions from Max and John, please send them a quick note thanking them: uhlenhuthm@gmail.com jroth.solar@gmail.com
+
+## Upcoming Meetup: Feb 26th at Akamai
+
+This month, Shankar Ambady from MassChallenge will be giving a talk on Natural Language Processing using [NLTK](http://www.nltk.org/) and Django. This should be awesome! Shankar has given a talk to Boston Python on NLTK, and now he's adding Django to the mix. RSVP [here](http://www.meetup.com/djangoboston/events/220373782/).
+
+We're still looking for a drink sponsor for the February meetup. If your company is interested in sponsoring, please let us know by responding to this email. Sponsorship will allow us all to go out for drinks together after the meetup, and gives you an opportunity to spread the word about your company and development team.
+
+## How Django Boston works
+
+- Meetups happen on the 4th Thursday of the month (always at Akamai in Cambridge, unless otherwise noted)
+- Pizza and salad is always provided (by Akamai)
+- We often migrate to Mead Hall for drinks afterward (sponsored by another company)
+- Everyone is welcome: beginners, experts, intermediate, not-sure-how-to-categorize-yourself. EVERYONE!
+
+## Call for speakers
+
+This is the part of the email that's supposed to pump you up about giving a talk. To do so, I'll give you a quick story from my childhood:
+
+In kindergarten, my teacher asked me to give a talk on my favorite animal. I knew what it was (flying squirrel) so I was all like, "yeah, I got this." Well, as it happened, I got up to give the talk, and my mind went TOTALLY blank. It was the first time I had experienced this phenomenon, so I freaked out and ran back to my desk, but before I got there, I slipped on Lucy's Jello, flipped onto my back, and ripped out the seat of my pants. Sadly, nobody got to hear about the flying squirrel, and everybody laughed at me.
+
+The moral of the story: No matter what happens during your talk, it can't be the worst talk ever, cause I own that. And your talk will likely be THE BEST ONE EVER.
+
+But all jokes aside: we're all relying on the community as we make our living using freely available tools, and free knowledge generously shared by previous speakers. Please seriously consider helping the community by giving a talk (and making yourself look cool and seem smart, which you are!).
+
+Of course, you can give a talk on any topic you want (it should be about Django, or at least something interesting for Django programmers). If you don't know what you want to give a talk on, there are many ideas below. If you don't know what to do, but feel the nudge to share your knowledge, respond to the email, and one of us will help you figure it out. If you don't want to give a talk, but are interested in something, let us know.
```

Comment:
Might also want to suggest giving a "lightning talk" which is only 5-10 min and might be a good way to test the waters before committing to a full 45 min talk.